### PR TITLE
AO3-6120 Add our IPs and secure URL to PERMITTED_HOSTS to hide proxy notice

### PR DESCRIPTION
**Pull request by zz9pzza**
# Pull Request Checklist

* [x] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [x] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [x] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [x] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [x] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6120

## Purpose

The warning should not occur on https://secure.archiveofourown.org/ 

## Testing Instructions

This can't be tested on staging, apart from noting that we have not got the warning on https://test.archiveofourown.org

### CODE REVIEW

File: config/config.yml [hunk 1]
```diff
@@ -460,6 +460,7 @@ PERMITTED_HOSTS: [
   "archiveofourown.org",
   "download.archiveofourown.org",
   "insecure.archiveofourown.org",
+  "secure.archiveofourown.org",
```
Comment (sarken) [inline config/config.yml:463]: Can we add the IP address mentioned in the issue too? Or is there a reason we shouldn't do that?

Comment (zz9pzza) [reply config/config.yml:463]: I am moving over to a new ip address before we balance over a new pair, I don't really support the bare ip address although we could for http but not https.

https://208.85.241.152/ is a security warning as expected. I could make http://208.85.241.152/  redirect to https:archiveofourown.org/ I think.

If I did that we still wouldn't want the ip address in it ?

Comment (sarken) [reply config/config.yml:463]: Support says people use the IP address to get around DNS poisoning. Would redirecting affect their ability to do that?

Comment (zz9pzza) [reply config/config.yml:463]: I will add the 2 ip's then.

**Review by sarken — APPROVED** [review]

File: config/config.yml (hunk 1 above)
Comment (sarken) [reply config/config.yml:466]: fwiw, I'm getting 503s on the IP addresses (using HTTP), but I assume you're aware of that

**Pull request merged**